solr/solrj/src/test/org/apache/solr/client/solrj/embedded/SolrExampleStreamingBinaryTest.java

```diff
@@ -35,9 +34,15 @@ public class SolrExampleStreamingBinaryTest extends SolrExampleStreamingTest {
 
   @Override
   public SolrClient createNewSolrClient() {
-    ConcurrentUpdateSolrClient client = (ConcurrentUpdateSolrClient) super.createNewSolrClient();
-    client.setParser(new BinaryResponseParser());
-    client.setRequestWriter(new BinaryRequestWriter());
+
+    SolrClient client =
+        new ErrorTrackingConcurrentUpdateSolrClient.Builder(getServerUrl())
+            .withQueueSize(2)
+            .withThreadCount(5)
```

Comment:
not needed

Reply:
can you elaborate on what is not needed?   Defining the client here versus just returning from the .build()?

Reply:
Queue size and thread count? They aren't in the original?

Reply:
looking into it now...   i might have goofed!  I remember thinking "this is different"...